src/components/ResponsiveImage.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { Image } from 'grommet'
+import styled, { css } from 'styled-components'
+import PropTypes from 'prop-types'
+
+export const StyledDiv = styled.div`
+  ${props => props.border && css`border: 1px solid black;`}
+`
+
+export const StyledImage = styled(Image)`
+  ${props => props.height && css`height: ${props.height};`}
+  max-width: 100%;
+`
+
+export default function ResponsiveImage({ border = false, height = 'auto', src }) {
+  return (
+    <StyledDiv border={border}>
+      <StyledImage height={height} fit='contain' src={src} />
+    </StyledDiv>
```

Comment:
There's a tiny gap at the bottom of the image:

<img width="395" alt="Screen Shot 2020-06-29 at 11 09 36 AM" src="https://user-images.githubusercontent.com/5016731/86031760-eea5e200-b9fb-11ea-9e9c-09479e58301b.png">